runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java

```diff
@@ -875,6 +894,25 @@ private void removeTzhaarTimer()
 		}
 	}
 
+	private void checkTeleport(VarPlayer varPlayer)
+	{
+		int lastTeleport = client.getVar(varPlayer);
+		long lastTeleportSeconds = (long) lastTeleport * 60;
+		boolean homeTeleport = varPlayer.equals(VarPlayer.LAST_HOME_TELEPORT);
+
+		Instant teleportExpireInstant = Instant.ofEpochSecond(lastTeleportSeconds).plus(homeTeleport ? 30 : 20, ChronoUnit.MINUTES);
```

Comment:
I don't think this calculation is needed. From my experience ingame, the teleport duration is not variable between 19-20 or 29-30 minutes. I suspect the transmitted var is epoch minutes rather than epoch seconds to reduce the size of the transmitted int, but is saved as epoch seconds on the server side.

Reply:
We'll have to disagree on this one. I tested this a few times (and once again after I read your review) and it seems correct.

Reply:
I can confirm it works the way you describe--it is in fact variable between 19-20 and 29-30 minutes respectively for minigame and home teleports.